_includes/sidebar.html

```diff
@@ -12,15 +12,22 @@
     <nav class="sidebar-nav">
       {% for sections in site.contents[page.lang] %}
         {% assign _section = sections[0] | strip %}
+        {% if page.section == _section %}
+          {% assign opened = true %}
+        {% elsif page.section == "home" and forloop.index0 == 1 %}
+          {% assign opened = true %}
```

Comment:
Thanks, @eksperimental!
Concerning 2 condition with the same result, I really expected to see that in the review, because this way is pretty obvious and I did not want to go against the DRY principles. But for some reason, a complex condition like this one, did not work, and I even was stuck looking at an old Jekyll [issue](https://github.com/Shopify/liquid/issues/138) on Github about boolean algebra. Now everything seems to be fine, I don't even know what could possibly go wrong and I feel a little ashamed.

The choice of `forloop.index0 == 1` instead of `_section == 'basics'` was motivated by a desire to leave one less thing to worry about in case if the sections will be changed someday (although, it feels like this will likely never happen).

I've just updated the branch. Is there anything else I could help with?